app/models/enterprise.rb

```diff
@@ -198,10 +188,6 @@ def self.distinct_count
     count(distinct: true)
   end
 
-  def activated?
-    confirmed_at.present? && sells != 'unspecified'
```

Comment:
@Matt-Yorkley, owners of all *new* enterprises will be confirmed, but we have to handle legacy data unfortunately...